lib/route/report/report.dart

```diff
@@ -266,6 +266,38 @@ class _ReportRouteState extends State<ReportRoute> with TickerProviderStateMixin
               refreshReport();
             },
           ),
+          (report != null)
+              ? Positioned(
+                  right: 4,
+                  top: context.padding.top + 4,
+                  child: Row(
+                    children: [
+                      Visibility(
+                        visible: report!.magnitude >= 6 && report!.magnitude < 7 && report!.getLocation().contains("海"),
+                        child: ActionChip(
+                          avatar: const Icon(Symbols.tsunami_rounded),
+                          label: const Text("可能引起若干海面變動"),
+                          backgroundColor: Colors.blue.withOpacity(0.16),
+                          labelStyle: const TextStyle(fontWeight: FontWeight.w900),
+                          side: const BorderSide(color: Colors.blue),
+                          onPressed: () {},
+                        ),
+                      ),
+                      Visibility(
+                        visible: report!.magnitude >= 7 && report!.getLocation().contains("海"),
+                        child: ActionChip(
```

Comment:
Don't use `ActionChip()`'s for display ppurpose, see [Material 3's Guidelines on Chips](https://m3.material.io/components/chips/guidelines) for a clearer reference on how  and when to use it.